connectivity/check/policy.go

```diff
@@ -146,7 +146,6 @@ func createOrUpdateKNP(ctx context.Context, client *k8s.Client, knp *networkingv
 	// Overload the field that should stay unchanged.
 	policy.ObjectMeta.Labels = knp.ObjectMeta.Labels
 	policy.Spec = knp.Spec
-	policy.Status = networkingv1.NetworkPolicyStatus{}
```

Comment:
what happened to this field

Reply:
Looks like it was removed upstream by https://github.com/kubernetes/kubernetes/pull/115843 ref. https://github.com/kubernetes/enhancements/issues/2943

Reply:
how disappointing 😿